Source/Thunder/Controller.cpp

```diff
@@ -1010,6 +1009,7 @@ namespace Plugin {
                 }
                 else {
                     result = stateControl->Request(PluginHost::IStateControl::command::RESUME);
+                    NotifySuspendResumeStateChange(callsign, Exchange::Controller::ILifeTime::state::RESUMED);
```

Comment:
I think this is too early, we can only do so if the state is indeed changed. You know that if you would subscribe to the IStateControl::INotification interface. What if the Suspend or Resume fails ?
Lets have a broader discussion on this if we want this as this has some severe impact..